met-web/src/components/engagement/form/EngagementWidgets/Phases/PhasesOptionCard.tsx

```diff
@@ -1,44 +1,52 @@
 import React, { useContext, useState } from 'react';
```

Comment:
I deleted a duplicated file and added a new one for phases, I'm not sure why it thinks I moved it